other/utils/twitter.js

```diff
@@ -0,0 +1,36 @@
+/* eslint camelcase:0 */
+
+import Twitter from 'twitter'
+
+const TW = new Twitter({
+  consumer_key: process.env.TW_CONSUMER_KEY,
+  consumer_secret: process.env.TW_CONSUMER_SECRET,
+  access_token_key: process.env.TW_ACCESS_TOKEN_KEY,
+  access_token_secret: process.env.TW_ACCESS_TOKEN_SECRET,
+})
+
+function getProfileImageURL(params) {
+  return new Promise((resolve, reject) => {
+    TW.get('users/show.json', params, (error, data) => {
+      if (!error) {
+        resolve(data.profile_image_url.replace('_normal', ''))
+      } else {
+        reject(error)
+      }
+    })
+  })
+}
+
+function sendTweet(message) {
```

Comment:
Thanks for adding this. I think I'll keep it, but I can't use it as is yet because the tweet should include the show screenshot and mentions of people on the show (like [this one](https://twitter.com/JavaScriptAir/status/718096907906584576)).